scout/server/blueprints/variants/views.py

```diff
@@ -123,7 +123,9 @@ def variants(institute_id, case_name):
     if request.form.get("export"):
         return controllers.download_variants(store, case_obj, variants_query)
 
-    data = controllers.variants(store, institute_obj, case_obj, variants_query, result_size, page)
+    data = controllers.variants(
+        store, institute_obj, case_obj, variants_query, result_size, page, query_form=form.data
```

Comment:
The entire form dictionary is passed to controllers even though only one key/value is used afterwards. I'm wondering if it wouldn't be easier just to pass the key/value in question (either a number or don't pass it at all?)

Reply:
Yeah, as above, it would certainly be easier now, though more complicated later. I can change if you like, and we just change it back later; either is fine by me!

Reply:
I would keep it easier now and change it later. I also think that when we change it later we should try to move much of the code present in the view to the controllers, so the code would probably be quite different from now!

Reply:
Right: it sure has some weirdness like direct store calls - and quite a bit of passing on the form to the controller. Landing more on the controller side would also make it easier to split the One View with multiple jobs into several with a single task.